.github/actions/build-docker-image-dockerhub/action.yml

```diff
@@ -0,0 +1,87 @@
+name: 'Build Docker image'
+description: 'Build Rocket.Chat Docker image'
+
+inputs:
+  root-dir:
+    required: true
+  docker-tag:
+    required: true
+  release:
+    required: true
+  username:
+    required: false
+  password:
+    required: false
+
+outputs:
+  image-name:
+    value: ${{ steps.build-image.outputs.image-name }}
+
+runs:
+  using: composite
+  steps:
+    # - shell: bash
```

Comment:
@huma0605 do we want to keep this? i'm curious why we command it out

Reply:
yeah, let me remove them.